website/src/pages/index.tsx

```
@@ -19,14 +19,30 @@
+                className="absolute -right-8 -top-30 w-22 rotate-6  z-10 animate-float"
+                src="img/3MBanner/heart-balloon.png"
+              />
+              <h1 className="title-font text-4xl leading-[3rem] lg:text-5xl lg:leading-[4rem] font-extrabold text-charcoal-300 dark:text-white mt-[80px] lg:w-3/4 mx-auto">
+                Best Free &amp; Open Source Tool for Containers &amp; Kubernetes
+              </h1>
+            </div>
+            <div className="relative">
+              <img
+                className="absolute left-[15px] -top-[150px] w-30 animate-float-diagonal"
+                src="img/3MBanner/seal-with-party-horn.png"
+              />
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)